package.json

```diff
@@ -8,15 +8,24 @@
   "scripts": {
     "storybook": "start-storybook"
   },
+  "jest": {
+    "testRegex": [
+      "spec/javascripts/.*_test\\.js"
+    ]
+  },
   "devDependencies": {
     "@storybook/addon-controls": "^6.3.7",
     "@storybook/addon-docs": "^6.3.7",
     "@storybook/server": "^6.3.7",
+    "babel-jest": "^27.0.6",
+    "babel-preset-es2015": "^6.24.1",
     "jasmine-core": "~2.4.1",
+    "jest": "^27.0.6",
     "karma": "~6.3.4",
     "karma-chrome-launcher": "~3.1.0",
     "karma-coffee-preprocessor": "~1.0.1",
     "karma-jasmine": "~0.3.8",
+    "mutationobserver-shim": "^0.3.7",
```

Comment:
Actually this one seems to be not needed (thanks for pointing me that out): since jsdom now supports MutationObserver: https://github.com/jsdom/jsdom/issues/639#issuecomment-816851899

Will remove the dep and then force push.